app/assets/javascripts/standalone/user_analytics.js

```diff
@@ -193,11 +193,11 @@ function filterDataByGender(tableName) {
 // Overlays
 //
 function openWindow(id, parentId) {
-  var element = document.getElementById(id);
+  let element = document.getElementById(id);
```

Comment:
Use ES6 syntax.

Reply:
Same question as above: How reliable is this in android webviews?